### PR TITLE
feat: email verification

### PR DESCRIPTION
**Pull request by siddhart1o1**

### CODE REVIEW

File: apps/dashboard/services/graphql/mutations/totp.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// import { gql } from "@apollo/client";
```
Comment (nicolasburtey) [inline apps/dashboard/services/graphql/mutations/totp.ts:1]: we don't want comment files committed in our repo

File: apps/dashboard/services/graphql/mutations/email.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      emailRegistrationId
+      errors {
+        message
+        code
+      }
+    }
+  }
+`;
+
+gql`
```
Comment (nicolasburtey) [inline apps/dashboard/services/graphql/mutations/email.ts:27]:
```suggestion
```

Comment (nicolasburtey) [reply apps/dashboard/services/graphql/mutations/email.ts:27]: you can combine queries/mutation in a single `gql` string

File: apps/dashboard/app/security/server-actions.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  const token = session?.accessToken;
+  if (!token && typeof token !== "string") {
+    return {
+      error: true,
+      message: "Invalid Token",
+      data: null,
+    };
+  }
+
+  let data;
```
Comment (nicolasburtey) [inline apps/dashboard/app/security/server-actions.ts:37]: no implicit any

**Comment by sandipndev** [comment]
I was able to delete the email and then re add it using the code from kratos. Is that what is intended in this PR?

**Review by sandipndev — APPROVED** [review]

**Pull request merged**